service/src/java/org/apache/hive/service/cli/session/HiveSessionImpl.java

```diff
@@ -408,18 +400,10 @@ private synchronized void acquireAfterOpLock(boolean userAccess) {
     // set the thread name with the logging prefix.
     sessionState.updateThreadName();
 
-    // If the thread local Hive is different from sessionHive, it means, the previous query execution in
-    // master thread has re-created Hive object due to changes in MS related configurations in sessionConf.
-    // So, it is necessary to reset sessionHive object based on new sessionConf. Here, we cannot,
-    // directly set sessionHive with thread local Hive because if the previous command was REPL LOAD, then
-    // the config changes lives only within command execution not in session level.
-    // So, the safer option is to invoke Hive.get() which decides if to reuse Thread local Hive or re-create it.
-    if (Hive.getThreadLocal() != sessionHive) {
-      try {
-        setSessionHive();
-      } catch (HiveSQLException e) {
-        throw new RuntimeException(e);
-      }
+    try {
```

Comment:
Not sure I understand this comment.  The "if" clause went away since we do not depend on anything with thread local storage to setSessionHive.  The format changed only because the "if" statement went away.  I guess I could leave it indented an extra 2 spaces...is that what you're suggesting?

Reply:
My bad. Did not see if block going away.